Makefile

```diff
@@ -113,9 +121,10 @@ endef
 # -------------------------------------------------------------------------------------------------
 $(OUT_DIR)/.config:
 ifndef BOARD
-	@echo "Variable BOARD must be defined to initialize output directory" >&2 && exit 1
```

Comment:
Why dropping exit here?

Reply:
Basically for simplification, if the BOARD variable is not defined it just should show the echo.
I tested this with: make $PWD/output/.config